### ISSUE TITLE
Please add 'Number' class to knownTypes for Jackson

### ISSUE THREAD

**Issue by rpsandiford** [stoicflame/enunciate#1095]
Java type 'Number' should be included in the knownTypes settings in EnunciateJacksonContext, probably as a KnownJsonType.WHOLE_NUMBER

**Comment by rpsandiford** [comment]
(Without this, the Enunciate debug output includes:

```
[DEBUG] [ENUNCIATE] Added java.lang.Number as a Jackson type definition.
[WARNING] [ENUNCIATE] Some source files were not found for the Java classes that define the Web service API.
[WARNING] [ENUNCIATE] Enunciate will be unable to find the documentation for the classes that are missing source files. (For details, rebuild with debug-level logging.)
[WARNING] [ENUNCIATE] In order for Enunciate to find the source files for API classes that are not being compiled in the current project, the source file must be on the Enunciate sourcepath and the class must be explicitly included in the Enunciate configuration.
[WARNING] [ENUNCIATE] For more information, see https://github.com/stoicflame/enunciate/wiki/Discovering-Source-Files
[DEBUG] [ENUNCIATE] Unable to find source file for java.lang.Number.
```

**Comment by stoicflame** [comment]
Released in `2.14.0`.
